tox.ini

```diff
@@ -5,11 +5,12 @@ envlist =
   py{27,35,36,37,38}-marshmallow2,
   py{35,36,37,38}-marshmallow3,
   py{36,37,38}-pydantic,
+  doctest,
+  coverage,
```

Comment:
If we're going to merge #356 coverage should be last since we're running in CI with two jobs.

Reply:
Maybe not? Because it seems that in parallel mode tox runs the evns unordered.

I guess we can exclude it and run it afterwards.